### ISSUE TITLE
Missing description of categories in README

### ISSUE THREAD

**Issue by mikapfl** [JGuetschow/PRIMAP-hist#69]
The following categories which are used in PRIMAP-hist 2.4.2 are missing in the README (on zenodo in the description and in the pdf):

* [ ] `0`
* [ ] `3`
* [ ] `M.LULUCF`

I think it would be good to describe all three. Only `3` is a standard IPCC2006 category, the other two are non-standard and deserve a proper definition. We can take the definition of `0` from climate_categories ("National Total"), I think, but the [description of `M.LULUCF`](https://github.com/pik-primap/climate_categories/blob/main/data_generation/IPCC2006_PRIMAP.py#L29) is not optimal there, we should probably find something a bit less circular.

Maybe we could also show the relationships of the used categories in a tree like the ones generated by climate_categories? Something like this:

```
In contrast to the standard IPCC 2006 categories, category 
3 "Agriculture, Forestry, and Other Land Use" is subdivided as follows:

3 Agriculture, Forestry, and Other Land Use
├M.AG Agriculture
│├3.A Livestock
│╰M.AG.ELV Agriculture excluding Livestock
╰M.LULUCF Land Use, Land Use Change, and Forestry

Where a national total is reported, it is calculated as the sum of
"M.0.EL Total emissions excluding LULUCF" and
"M.LULUCF Land Use, Land Use Change, and Forestry":

0 National Total
├M.0.EL Total emissions excluding LULUCF
│├1 Energy
│├2 Industrial Processes and Product Use
│├4 Waste
│├5 Other
│╰M.AG Agriculture
╰M.LULUCF Land Use, Land Use Change, and Forestry

```

It would probably help to understand the relationships (especially for people who aren't used to IPCC 1996 categories).

**Comment by JGuetschow** [comment]
That's sort of on purpose. These categories are only included in the no-rounding files that are only targeted at expert users. But probably it's best to include them but in an individual table to make sure people are not looking for them in the standard version.
The idea with the relationship trees is good

**Comment by JGuetschow** [comment]
I've added an additional table with the extra categories and the tree descriptions of the categories to the data description. The changes will be included in the data description of v2.5
